e2e/cypress.json

```diff
@@ -1,5 +1,6 @@
 {
     "baseUrl": "http://localhost:8065",
+    "webhookBaseUrl": "http://localhost:3000",
```

Comment:
I think we should only put valid Cypress settings in this file so it's clear that these options are for configuring cypress. We can use the `env` section of this file for variables like this, IMO.

Reply:
I agree, see updated commit.  Thanks